### ISSUE TITLE
Pascal VOC with split_0, 1/16 setting get 55.17 mAP?

### ISSUE THREAD

**Issue by djh-dzxw** [LiheYoung/ST-PlusPlus#18]
We use your code to reproduce the experimental results in your paper. 
Our settings are pascal_batchsize_16_resnet50_deeplabv3plus_pascal_1_16_split_0
but get 55.17mAP
In your paper, it's about 73 mAP with ST++

**Comment by LiheYoung** [comment]
We use MeanIoU as an evaluation metric. 

I can successfully reproduce our results with this codebase, also supported [here](https://github.com/LiheYoung/ST-PlusPlus/issues/15). Did you correctly load the pre-trained backbones?

**Comment by djh-dzxw** [comment]
> 
We have found the reason of the accuracy gap, which is we use 8 V100s with the batchsize of 16 in Pascal experiment. I suppose that you leverage BN as the regularization of the backbone, which distribute a batch of data to GPUs. We are making a new experiment on two V100s, in which everything is temporarily OK.

**Comment by LiheYoung** [comment]
Yes, two GPUs will be better, since we do not apply SyncBN. Please feel free to contact us if you have further questions.
